cruise/CruiseInformation.csv

```diff
@@ -282,4 +282,4 @@ PS20200917,R/V Pacific Storm,2020-09-17T17:30:00,2020-09-20T17:30:00,Endurance C
 TS20200925,F/V Tenacious II,2020-09-25T21:00:00,2020-09-26T15:00:00,Pioneer 14c Glider deployments
 ZP20200930,R/V Zephyr,2020-09-30T17:00:00,2020-10-03T23:59:59,Papa 7a Glider deployments
 EN657,R/V Endeavor,2020-10-13T13:00:00,2020-10-18T16:00:00,Pioneer glider cp_340 recovery
-AR48,R/V Neil Armstrong,2020-10-28T00:00:00,2020-11-11T23:59:00,Pioneer 15
+AR48,R/V Neil Armstrong,2020-10-28T13:34:00,2020-11-11T16:36:00,Pioneer 15
```

Comment:
Looks good.

Reply:
Looks good

Reply:
Looks good (Jennifer just beat me to it!)